### ISSUE TITLE
Draco not recognized in 2.0.0

### ISSUE THREAD

**Issue by camnewnham** [atteneder/glTFast#84]
I'm getting the error `DracoUnity package needs to be installed in order to support glTF extension KHR_draco_mesh_compression` when following the instructions to install this library and subsequently Draco support in Unity 2020.1.5f1.

From a glance at the code it looks like `DRACO_UNITY` is not getting defined properly. Digging a bit further I found that the asmdef for glTFastis asking for version **1.2.0**:
```
"name": "com.atteneder.draco",
            "expression": "1.2.0",
            "define": "DRACO_UNITY"
```
whereas https://gitlab.com/atteneder/DracoUnity/-/blob/master/package.json has the latest as **1.1.2**. (This is also apparent in the release .zip)

Manually defining `DRACO_UNITY`  removes the immediate error, but results in further errors:

```
Library\PackageCache\com.atteneder.gltfast@b1176da67a\Runtime\Scripts\GltFast.cs(1606,17): error CS1061: 'DracoMeshLoader.DracoJob' does not contain a definition for 'weightsId' and no accessible extension method 'weightsId' accepting a first argument of type 'DracoMeshLoader.DracoJob' could be found (are you missing a using directive or an assembly reference?)

Library\PackageCache\com.atteneder.gltfast@b1176da67a\Runtime\Scripts\GltFast.cs(1607,11): error CS1061: 'DracoMeshLoader.DracoJob' does not contain a definition for 'jointsId' and no accessible extension method 'jointsId' accepting a first argument of type 'DracoMeshLoader.DracoJob' could be found (are you missing a using directive or an assembly reference?)
```

Is there a later version of DracoUnity that I'm missing for 2.0.0 compatibililty?

**Comment by atteneder** [comment]
Thanks for reporting!

Seems I messed up by not doing a DracoUnity Release :/

I just released DracoUnity 1.3.0 (with weight/joint support) to fix it.

Please test, let me know and close the issue in case it worked.

**Comment by camnewnham** [comment]
Thanks for the prompt fix Andreas! I can confirm that this issue is resolved.
